web-ui/src/components/reviews/TeamReviews.jsx

```diff
@@ -37,47 +47,55 @@ import {
   selectIsAdmin,
   selectMyTeam,
   selectCurrentMembers,
-  selectSubordinates,
-} from "../../context/selectors";
-import { getAvatarURL } from "../../api/api.js";
-import DateFnsUtils from "@date-io/date-fns";
+  selectSubordinates
+} from '../../context/selectors';
+import {getAvatarURL} from '../../api/api.js';
+import DateFnsUtils from '@date-io/date-fns';
 const dateUtils = new DateFnsUtils();
 
 const propTypes = {
-  teamMembers: PropTypes.arrayOf(PropTypes.shape({id: PropTypes.string, firstName: PropTypes.string, lastName: PropTypes.string,})),
-  periodId: PropTypes.string,
+  teamMembers: PropTypes.arrayOf(
+    PropTypes.shape({
+      id: PropTypes.string,
+      firstName: PropTypes.string,
+      lastName: PropTypes.string
+    })
+  ),
+  periodId: PropTypes.string
 };
-const displayName = "TeamReviews";
+const displayName = 'TeamReviews';
 
 const PREFIX = displayName;
 const classes = {
   actionButtons: `${PREFIX}-actionButtons`,
   headerContainer: `${PREFIX}-headerContainer`,
-  periodModal: `${PREFIX}-periodModal`,
+  periodModal: `${PREFIX}-periodModal`
 };
 
 const Root = styled('div')(({theme}) => ({
   [`& .${classes.actionButtons}`]: {
-    margin: "0.5em 0 0 1em",
-    ['@media (max-width:820px)']: { // eslint-disable-line no-useless-computed-key
-      padding: "0",
-    },
+    margin: '0.5em 0 0 1em',
+    ['@media (max-width:820px)']: {
+      // eslint-disable-line no-useless-computed-key
+      padding: '0'
+    }
   },
   [`& .${classes.headerContainer}`]: {
-    display: "flex",
-    'flex-direction': "row",
```

Comment:
I changed all CSS properties that do not use camelCase names to use them in order to remove warnings.